### ISSUE TITLE
Something about the end token predict

### ISSUE THREAD

**Issue by sophieag** [Russell91/nlpcaffe#20]
Hi, I'm working with the nlp-caffe to replicate the neuraltalk code, to finish a video description task. And I find I just cannot predict the end token. It seems that it will predict all the words to the max length. Like below:

uturn butts preparation advancing costmetics travelers preparation preparation slcied slcied slcied slcied slcied slcied slcied slcied slcied slcied slcied slcied

Have you ever encountered with this? Could you give me some advices about it? 
Thank you so much~

**Comment by Russell91** [comment]
Yes, the code does not insert end of sentence symbols automatically, so you will need to do this yourself. When you are tokenizing your file, be sure to add an <eos> symbol. The network will predict this just like any other word, and then fill the rest of the sentence with pad symbols.

**Comment by sophieag** [comment]
Well, I have checked my lmdb data, and the end token and zero symbols are all included. It just seems that the network did not get well trained? Should the solutions be adding some data or try different solver settings? Could you give me some advice?
